hue/templates/configmap.yaml

```diff
@@ -43,6 +43,22 @@ data:
         # Database schema, to be used only when public schema is revoked in postgres
         ## schema=public
 
+    
+    # Database Connections inside the Editor
+
+    [librdbms]
+      [[databases]]
```

Comment:
Are these two lines needed if there are no databases enabled?

Reply:
Not needed but they don't hurt either... The idea was to leave those outside the postgres conditional in case we want to add any other rdbms later.

Reply:
I know, but what happens if there are no databases at all?

Reply:
Nothing, this section is used just to declare the rdbms connections. It then has to be used in the  interpreters section. So if nothing is declared and nothing is used in the interpreters there won't be any problem.